tools/init_databases.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python3
+# Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+# For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+# import required module
+import os
+import sqlite3
+import argparse
+
+
+# this helper script creates DBs from SQL schema files
+
+def process_file(path: os.path, dst_directory: os.path) -> None:
```

Comment:
meaningless function name. Process what? Process for what?

Reply:
fixed

Reply:
Ok